core/modules/qdisp/QueryRequest.cc

```diff
@@ -33,6 +33,7 @@
 #include "qdisp/QueryRequest.h"
 
 // System headers
+#include <cstddef>
```

Comment:
I don't see any nullptr changes elsewhere in this file -- is this #include still needed, or was the NULL usage factored away before you rebased?